CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ This changelog only contains changes starting from version 1.3.0
 
 ## Compiler settings
```

Comment:
I would add a changelog item that we are renaming the repository.

Reply:
I think we should mention it as part of 1.5 (since we haven't released it yet and would affect that version).

Reply:
> I think we should mention it as part of 1.5 (since we haven't released it yet and would affect that version).

Done here: https://github.com/safe-global/safe-contracts/pull/733/commits/f046cb9049b001fba914fedb359556008cd537ea